### PR TITLE
Danielle baxter

### PR DESCRIPTION
**Pull request by danibaxx**
Submitting PR for PM review.

### CODE REVIEW

File: assignments/lambda-classes.js [hunk 1]
```diff
@@ -1 +1,149 @@
...
+  PRAssignment(subject) {
+    console.log(`${this.name} has submitted a PR for ${subject}.`);
+  }
+
+  sprintChallenge(subject) {
+    console.log(`${this.name} has begun sprint challenge on ${subject}.`);
+  }
+}
+
+const subject = 'JavaScript Fundamentals';
```
Comment (ari7946) [inline assignments/lambda-classes.js:78]: const variable *subject* is not needed here. The subject can be passed as an argument to the methods above when invoking them.

File: assignments/lambda-classes.js [hunk 2]
```diff
@@ -1 +1,149 @@
...
+console.log(bella.listsSubjects());
+console.log(harry.PRAssignment('React'));
+console.log(bella.sprintChallenge('JavaScript Fundamentals'));
+
+console.log(severus.speak());
+console.log(severus.debugsCode(harry, 'JavaScript'));
+console.log(ginny.speak());
+console.log(ginny.standUp('#webPT6'));
+console.log(ginny.demo('Python'));
+console.log(severus.grade(bella));
```
Comment (ari7946) [inline assignments/lambda-classes.js:149]: The subject can be passed down as an argument along with the student.

**Review by ari7946 — APPROVED** [review]
Nice Job, Danielle! I'm really like the Harry Potter theme.

File: assignments/lambda-classes.js (hunk 1 above)
Comment (danibaxx) [reply assignments/lambda-classes.js:78]: I left that in by accident, oopse. :D

**Pull request merged**